.changeset/proud-houses-glow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": minor
+---
+
+removed the “isDisabled“ prop from the “Button” component in favour of the native “disabled“ attribute
```

Comment:
[issue] this is misleading, since that's what it already did (provided the native disabled attribute). suggestion: 

```suggestion
removed the “isDisabled“ prop from the “Button” component; added instructions for developers to manually add it themselves if needed.
```